build/NuGetPackages/Microsoft.Build.Runtime.nuspec

```diff
@@ -19,12 +19,14 @@
         <dependency id="Microsoft.Build.Framework" version="[$version$]" />
         <dependency id="Microsoft.Build.Tasks.Core" version="[$version$]" />
         <dependency id="Microsoft.Build.Utilities.Core" version="[$version$]" />
+        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.0.0" />
```

Comment:
You shouldn't need to add this to our runtime package because the dependency on Microsoft.Build transitively depends on `System.Runtime.InteropServices.RuntimeInformation`.